src/NuGet.Clients/NuGet.PackageManagement.UI/Utility/PackageSourceMappingUtility.cs

```diff
@@ -251,6 +266,11 @@ internal static class PackageSourceMappingUtility
                     newSourceMappings.Add(sourceNameToMap, new SortedSet<string>(new List<string>(capacity: added.Count) { addedPackage.Id }));
                 }
             }
+
+            if (firstLogError != null)
+            {
+                throw new ApplicationException(firstLogError.Message);
```

Comment:
Throwing here whatever the first error was which serves as
1. A way to bail out of the package installation.
2. When debugging, the `NU1110` will be readily available in the stack.